### PR TITLE
DOC: Add docs for EMPeror in Fig1

### PR DESCRIPTION
**Pull request by ElDeveloper**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,11 @@ loading large files in certain cases (for example Figure 1A-B). See
     1. Copy `TACGTAGGTCCCGAGCGTTGTCCGGATTTATTGGGCGTAAAGCGAGCGCAGACGGTTACTTAAGCAGGATGTGAAATCCCCGGGCTCAAC` into search bar
     2. Click `Search`
 
+After configuring the tree, you can now configure the ordination panel on the right:
+
+1. Go to the `Color` tab on the right and select `empo_1`.
```
Comment (fedarko) [inline README.md]:
```suggestion
1. Go to the `Color` tab on the right and select `empo_1` as the `Color Category` (in the first dropdown menu).
```

File: README.md [hunk 2]
```diff
@@ -30,6 +30,11 @@ loading large files in certain cases (for example Figure 1A-B). See
...
+2. In the second dropdown menu in the `Color` tab select `Yellow-Orange-Red`.
```
Comment (fedarko) [inline README.md]:
```suggestion
2. In the second dropdown menu in the `Color` tab, select `Yellow-Orange-Red`.
```

**Review by fedarko — COMMENTED** [review]
Looks good to me! Enclosing two very small changes to the wording that might make this clearer (up to you).

It is pretty wild seeing this much data at once :D

dang

**Comment by ElDeveloper** [comment]
Great, thanks so much @fedarko!

**Comment by fedarko** [comment]
Thanks @ElDeveloper!

**Pull request merged**